src/main/features/dashboard/views/macro/caseStatus.njk

```diff
@@ -175,7 +175,11 @@
   {% elif status === ClaimStatus.PART_ADMIT_PAY_IMMEDIATELY %}
     {{ partAdmittedClaimantAcceptsClaimantDashboard(claim) }}
   {% elif status === ClaimStatus.CLAIMANT_REJECTED_DEFENDANT_DEFENCE_NO_DQ %}
```

Comment:
in claimStatus I cas see 'CLAIMANT_REJECTED_DEFENDANT_DEFENCE_NO_DQ' is been removed, do we still need this check?